src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1598,6 +1598,16 @@ public Task Continue(DebuggedThread thread)
             return Execute(thread);
         }
 
+        public async Task Jump(string filename, int line)
+        {
+            await MICommandFactory.ExecJump(filename, line);
+        }
```

Comment:
I think we want to remove all the overloads that use filename and line number since we always seem to use the overload that take the address.